packages/api/tests/endpoints/granules/helpers.js

```diff
@@ -17,34 +14,37 @@ const { getBucketsConfigKey } = require('@cumulus/common/stack');
 
 const { getDistributionBucketMapKey } = require('@cumulus/distribution-utils');
 
-const {
-  fakeFileFactory,
-  fakeGranuleFactoryV2,
-} = require('../../../lib/testUtils');
+const getPgFilesFromGranuleCumulusId = async (knex, filePgModel, postgresGranuleCumulusId) =>
+  await filePgModel.search(knex, {
+    granule_cumulus_id: postgresGranuleCumulusId,
+  });
 
-const getPostgresFilesInOrder = async (knex, newGranule, filePgModel, postgresGranuleCumulusId) => {
-  const pgFiles = await Promise.all(newGranule.files.map(async (file) => {
-    const [pgFile] = await filePgModel.search(knex, {
-      granule_cumulus_id: postgresGranuleCumulusId,
-      file_name: file.fileName,
-    });
-    return pgFile;
-  }));
-  return pgFiles;
-};
+const getFileNameFromKey = (key) => key.match('[^\/]+$')[0];
```

Comment:
Hmm, it's a test helper, but surely this regex is encapsulated in a tested library somewhere?

Reply:
Oh sure. I was thinking `split` wouldn't work because we could have a `key` without a `/` but just tested and it works fine. 

https://github.com/nasa/cumulus/pull/2415/commits/c369ff6fc0f3288b8366f87d956d7a53cdd6c9c0